src/main/java/gov/nasa/jpl/mbee/mdk/mms/validation/ElementValidator.java

```diff
@@ -151,11 +149,12 @@ private void processServerElements(Map<String, Pair<Element, ObjectNode>> client
                     // solves edge case where first model validation incorrectly removes bins from project
                     removeServerObjectNodeUsingIdPrefix(elementObjects, MDKConstants.HOLDING_BIN_ID_PREFIX);
                 }
+                //Remove View Instances Bin
                 removeServerObjectNodeUsingIdPrefix(elementObjects, MDKConstants.VIEW_INSTANCES_BIN_PREFIX);
 
                 for(ObjectNode jsonObject : elementObjects) {
                     JsonNode idValue = jsonObject.get(MDKConstants.ID_KEY);
-                    if(idValue != null && idValue.isTextual() && !serverElementMap.containsKey(idValue.asText())) {
+                    if(idValue != null && idValue.isTextual() && !idValue.asText().startsWith(MDKConstants.HIDDEN_ID_PREFIX) && !serverElementMap.containsKey(idValue.asText())) {
```

Comment:
Based on further investigation and discussion at the dev tag-up, `hidden_` elements are only now popping up because of a local experiment to try using the server's `getAllElements` instead of a recursive search for performance reasons. This causes `hidden_` elements in the `View Instances Bin` and the `Holding Bin` to be returned while the recursive search method would/does not. Since the resolution to the performance issues is being addressed server-side, the recursive search can stick around and this change should no longer be needed.